qutip/sesolve.py

```diff
@@ -56,64 +56,62 @@
 def sesolve(H, psi0, tlist, e_ops=None, args=None, options=None,
             progress_bar=None, _safe_mode=True):
     """
-    Schrodinger equation evolution of a state vector or unitary matrix
-    for a given Hamiltonian.
+    Schrodinger equation evolution of a state vector or unitary matrix for a
```

Comment:
```suggestion
    Schrödinger equation evolution of a state vector or unitary matrix for a
```
Just for the fun of it. :)